### ISSUE TITLE
Reorder tabs to put Table first

### ISSUE THREAD

**Issue by Norric1Admin** [ONSdigital/sdg-indicators#3098]
Although the chart and map presentation of data is difficult to use or inaccessible to some user groups, an alternative presentation is available in a table. However, we would recommend presenting the table before the chart and map to benefit those user groups who are unable to access the information or elements within the chart and map facility.

As so much information can be displayed in the table at once which can be difficult for some users to navigate, consider dividing the information into multiple smaller tables or generating a table after users have selected the filter options.

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility tickets](https://trello.com/c/E40VA79G/549-accessibility-tickets)

**Comment by LucyGwilliamAdmin** [comment]
## Description

## Screenshots
**Before**

**After**


## Successfully tested on...
- [] Windows: IE11
- [] Windows: Chrome
- [ ] Windows: Firefox
- [ ] Windows: Edge
- [ ] macOS: Chrome
- [ ] macOS: Firefox
- [ ] iOS: Safari 9.3
- [ ] iOS: Chrome
- [ ] Samsung: Chrome
- [ ] Samsung: Samsung Internet

## User feedback and problems encountered...

**Comment by Norric1Admin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add `title` field to embedded content](https://trello.com/c/Af9LfuUh/607-add-title-field-to-embedded-content)

**Comment by AnnCorp** [comment]
Resource focussed on achieving AA, so won't be actioning AAA fixes currently.
